.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,29 @@
 ## What's new in this PR
+
```

Comment:
Can you create a file `.prettierignore`, add `.github/`,  and revert the changes to this file? We don't want the spacing changes it makes.